py3status/modules/imap.py

```diff
@@ -7,7 +7,7 @@
     cache_timeout: how often to run this check (default 60)
     criterion: status of emails to check for (default 'UNSEEN')
     format: format to display (default 'Mail: {unseen}')
-    hide_if_zero: don't show on bar if False (default False)
+    hide_if_zero: don't show on bar if True (default False)
```

Comment:
Good catch :)

Usually it'd be good to do this as a different PR but it's not that important so you might as well leave it here.